clang/include/clang/Sema/Sema.h

```diff
@@ -1909,7 +1909,22 @@ class Sema final : public SemaBase {
   /// '\#pragma clang attribute push' directives to the given declaration.
   void AddPragmaAttributes(Scope *S, Decl *D);
 
-  void PrintPragmaAttributeInstantiationPoint();
+  using DiagFuncRef =
+      llvm::function_ref<void(SourceLocation, PartialDiagnostic)>;
+  auto getDefaultDiagFunc() {
+    return [this](SourceLocation Loc, PartialDiagnostic PD) {
+      // This bypasses a lof of the filters in the diag engine, as it's
```

Comment:
```suggestion
      // This bypasses a lot of the filters in the diag engine, as it's
```

Reply:
Not part of this PR, this is currently not rebased.
This has been fixed in the parent PR, and will disappear when rebased.